README.md

```diff
@@ -788,14 +789,21 @@ scala> val low = 40.dollars / megawattHour
 low: squants.market.Price[squants.energy.Energy] = 40.00 USD/1.0 MWh
 ```
 
-Implicit conversion support for using Double on the left side of multiplication:
+Implicit conversion support for using Doubles, Longs and BigDecimals on the left side of multiply and divide operations:
```

Comment:
It looks like you edited `README.md` directly. The readme is built via tut now, so these changes will be overwritten on the next build. Can you also update `shared/src/main/tut/README.md` so they're preserved?